import.py

```diff
@@ -17,7 +20,7 @@
                     help='The Cookie header to pass to the Drupal server.')
 parser.add_argument('-cid', '--collection_id',
                     help='The Collection ID to import into.')
-parser.add_argument('-l', '--logfile', default=None,
+parser.add_argument('-l', '--logfile', default='../logs/bii_{}.log'.format(nows),
```

Comment:
@ThanGrove it might be nicer to define a constant in importers/base like:
```
import os
...
LOG_DIR = os.path.join(os.getcwd(), 'logs')
```
And to have this just
`default = 'bii_{}.log'.format(nows)`
and in imports base
`logging.basicConfig(filename=os.path.join(LOG_DIR, self.logfile), level=logging.DEBUG)`

Mostly I say this because later you are dumping a JSON of the metadata to a log dir, and it would be nice to refer to the '../logs' dir in a consistent fashion.

Reply:
This code was added to the base.py with a slight modification to implement the next suggestion. I have posted the code to the branch. I also included some bug fixes after testing the import.